pipeline_dp/dp_computations.py

```diff
@@ -759,3 +758,61 @@ def compute_sensitivities_for_normalized_sum(
     linf_sensitivity = max_abs_value * params.max_contributions_per_partition
 
     return Sensitivities(l0=l0_sensitivity, linf=linf_sensitivity)
+
+
+class ThresholdingMechanism:
+    """Performs partition selection with thresholding mechanism.
+
+    The (Laplace, Gaussian) thresholding algorithm is the following:
```

Comment:
I see, thanks! I am leaving this up to you, but as an API user, I would like to know how much delta will be used for threshold do that I can predict the threshold value.

Reply:
This class isn't public API for clients, it's part of the implementation. For the API of this class isn't important, how delta split, since it's not responsibitity of this class, but the responsibility of C++ class. For PipelineDP clients this information can be userful, and it's available in explain computation reports.